torch/lib/THPP/tensors/generic/THTensor.cpp

```diff
@@ -89,6 +89,30 @@ auto THTensor<real>::newUnfold(int dimension, long size, long step) const -> THT
 }
 
 template<>
+auto THTensor<real>::newExpand(const long_range& size) const -> THTensor* {
+  THLongStorage *size_storage = THLongStorage_newWithSize(size.size());
+  long *size_storage_d = size_storage->data;
+  for (auto it = size.begin(); it != size.end(); ++it)
+    *size_storage_d++ = *it;
```

Comment:
Isn't this `std::memcpy(size_storage->data, size.data(), sizeof(long) * size.length())`?